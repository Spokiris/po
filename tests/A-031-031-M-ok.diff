*** tests/A-031-031-M-ok.out	2023-10-19 17:08:13.000000000 +0100
--- tests/A-031-031-M-ok.outhyp	2023-10-27 19:48:41.616886176 +0100
***************
*** 25,31 ****
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Função a procurar: 2;3|48=ADD(2;2,5)
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
--- 25,32 ----
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Função a procurar: 2;3|48=ADD(=2;2,5)
! 
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
