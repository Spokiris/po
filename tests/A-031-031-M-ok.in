4
0
5
2
ADD
0
0
