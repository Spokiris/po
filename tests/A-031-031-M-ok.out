Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 4
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 5
Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: 2
Função a procurar: ADD
2;3|48=ADD(2;2,5)
Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 0
